src/wallet/rpcdump.cpp

```diff
@@ -1282,33 +1287,46 @@ static UniValue ProcessImport(CWallet * const pwallet, const UniValue& data, con
             }
              const CPubKey& pubkey = entry->second;
              CPubKey temp;
-             if (!pwallet->GetPubKey(id, temp) && !pwallet->AddWatchOnly(GetScriptForRawPubKey(pubkey), timestamp)) {
+             if (!pwallet->GetPubKey(id, temp) && !pwallet->AddWatchOnlyWithDB(*batch, GetScriptForRawPubKey(pubkey), timestamp)) {
```

Comment:
There is a weird indentation glitch here, although it looks like it was preexisting.

Reply:
What are you referring to? I don't see it.

Reply:
Ah, think I see it now - 1283-1287 are off by one.